src/components/Guestbook.js

```diff
@@ -1,178 +1,46 @@
-                    sx={{
-                        mb: 2,
-                        '& .MuiOutlinedInput-root': {
-                            '& fieldset': {
-                                borderColor: 'white',
-                            },
-                            '&:hover fieldset': {
-                                borderColor: 'white',
-                            },
-                            '&.Mui-focused fieldset': {
-                                borderColor: 'white',
-                            },
-                        },
-                        '& .MuiInputBase-input': {
-                            color: 'white',
-                        },
-                        '& .MuiInputLabel-root': {
-                            color: 'white',
-                        },
-                    }}
-                    inputProps={{maxLength: 500, color: 'white'}}
-                    helperText={`${message.length}/500`}
-                    FormHelperTextProps={{
-                        sx: {
-                            color: 'white',
-                        }
-                    }}
-                />
-                <Button variant="contained" color="primary" type="submit" fullWidth
-                        sx={{color: '#000428', borderRadius: 0}}>Write</Button>
-            </Box>
-            {entries.map((entry) => (
-                <Card key={entry.id} sx={{mb: 2, borderRadius: 0, width: '100%'}}>
-                    <CardContent sx={{
-                        padding: '4px',
-                        backgroundColor: theme.palette.spotifyDarkGrey.main,
-                        color: 'white',
-                    }}>
-                        <Typography variant="h6" align="center">{entry.name}</Typography>
-                        <Typography variant="body2" color="white"
-                                    align="center">{formatDateTime(entry.date)}</Typography>
-                        <Typography variant="body1" align="center">{entry.message}</Typography>
-                    </CardContent>
-                </Card>
-            ))}
-            {totalPages > 1 && (
-                <Pagination count={totalPages} page={page} onChange={handlePageChange} color="primary"
-                            sx={{justifyContent: 'center', display: 'flex', my: 2}}/>
-            )}
-        </Container>
-    );
+const Guestbook = () => {
+	const [page, setPage] = useState(1);
+
+	const { data: entries, isLoading } = useGetGuestbookEntries({
+		page: page - 1,
+		limit: entriesPerPage,
+	});
+
+	const handlePageChange = (_event, value) => setPage(value);
+
+	return (
+		<Stack sx={{ alignItems: "center" }}>
+			<img
+				src="/guestbook_500x200.gif"
+				alt="Guestbook Banner"
+				style={{ width: "60%", height: "auto", flexShrink: 0, padding: 8 }}
+			/>
+			<GuestbookWriter />
+			{isLoading && <Loading />}
+			{entries?.content?.map((entry) => (
+				<GuestbookEntry key={entry.id} entry={entry} />
```

Comment:
Ka lihtne ja mõnna asi mida teha, copy-paste see üks entry eraldi componenti. Palju lihtsam ja mõnusam lugeda ja kui tahad muuta siis leiad kiiresti üles kui vajutad ctrl + o ja kirjutada ent. Jetbrainsil ilmselt teine shortcut aga you get the pic.